boards/opentitan/earlgrey-cw310/Makefile

```diff
@@ -101,9 +103,7 @@ endif
 	mkdir -p $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/deps/
 	$(Q)cp test_layout.ld $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/deps/layout.ld
 	$(Q)cp ../../kernel_layout.ld $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/
-	$(Q)RUSTFLAGS="$(RUSTC_FLAGS_TOCK)" QEMU_ENTRY_POINT=${QEMU_ENTRY_POINT} $(CARGO) test $(CARGO_FLAGS_TOCK) $(NO_RUN) --bin $(PLATFORM) --release
-# 	Test layout should be removed so that following apps (non-test) load the correct layout.
-	$(Q)rm $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/deps/layout.ld
```

Comment:
This isn't needed?

Reply:
run.sh always contains the code to delete the file. It's currently only needed because without TOCK_ROOT_DIRECTORY and TARGET being passed, the rm in run.sh failed.